### PR TITLE
Update Contributing guide

### PR DESCRIPTION
**Pull request by Sekhmet**
Added few instructions that should make contributing easier for others.

Current document is there: https://docs.google.com/document/d/1tsR8ZOIF3Inf-KkZxZA3b0Q60Dct3Y0TmPhfuE5zbaA/edit#

Readme: https://github.com/busyorg/busy/blob/s-contributing/README.md
Contributing guide: https://github.com/busyorg/busy/blob/s-contributing/docs/CONTRIBUTING.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,28 +2,32 @@
 [![Crowdin](http://d322cqt584bo4o.cloudfront.net/busy/localized.svg)](https://translate.busy.org/project/busy)
 [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/busyorg/busy/new-design/LICENSE)
 [![Busy channel on Discord](https://img.shields.io/badge/chat-discord-738bd7.svg)](https://discord.gg/G95rNZs)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
```
Comment (bonustrack) [inline README.md]: We should use `https://img.shields.io/badge/PRs-welcome-brightgreen.svg` (without the flat-square style) if you want the same style everywhere

File: README.md [hunk 2]
```diff
@@ -2,28 +2,32 @@
...
-yarn install
+yarn
 yarn dev-server
 ```
 
-## Contribution 
-So you found a bug? Or perhaps you just want to improve something? Patches are welcome! If you would like to contribute, but don't know what to work on, check the issues list.
+## Getting involved
+
+We at Busy really appreciate help from the community. We want to make a Busy project as friendly too contributors as possible. It doesn't matter if you are a beginner looking for a place to improve your skills or just want to help us build something great.
```
Comment (jm90m) [inline README.md]: The 'a' here should be replaced in this sentence as well as some rephrasing of the sentence.`We want to make a Busy project as friendly too contributors as possible.` -> should be -> `We want to make the Busy project as friendly as possible to contributors.`

**Comment by bonustrack** [comment]
The google doc is great, well done! What do you think if we add a link to the guide when someone create an issue or PR. With these files template:
https://github.com/busyorg/busy/edit/master/docs/ISSUE_TEMPLATE.md
https://github.com/busyorg/busy/edit/master/docs/PULL_REQUEST_TEMPLATE.md

**Comment by Sekhmet** [comment]
I updated the document with few new sections. It should be good for a review/merge to make it more accessible.

**Pull request merged**